### PR TITLE
Staging/ideaspace

### PR DESCRIPTION
**Pull request by dbradham**

### CODE REVIEW

**Review by RETprojects — APPROVED** [review]

**Comment by dbradham** [comment]
@dev-launchers/platform-enablement when you get a chance, we would appreciate your review here!

File: packages/UI/src/components/atoms/Radio/StyledRadio.ts [hunk 1]
```diff
@@ -16,13 +16,15 @@ export const Option = styled.label`
...
     pointer-events: none;
   }
+
+  margin-bottom: 0.25rem;
 `;
 
 export const Input = styled.input`
   height: 24px;
-  width: 24px;
+  /* width: 24px; */
```
Comment (chungthuang) [inline packages/UI/src/components/atoms/Radio/StyledRadio.ts:25]: Why is this commented out?

File: apps/ideaspace/src/components/modules/SubmissionForm/SubmissionForm.js [hunk 2]
```diff
@@ -32,13 +32,13 @@ function SubmissionForm() {
   );
 
   const getDBInvolveLevel = (involveString) => {
-    if (involveString.includes('and also believe')) {
+    if (involveString.includes('to end')) {
```
Comment (chungthuang) [inline apps/ideaspace/src/components/modules/SubmissionForm/SubmissionForm.js:35]: This looks hacky. Can we change the backend to store this as an enum?

File: apps/ideaspace/src/components/common/IdeaForm/IdeaForm.js [hunk 3]
```diff
@@ -7,227 +7,236 @@ import SubmissionButton from './SubmissionButton';
...
-				enableReinitialize
-			>
-				{({ errors, setFieldValue, touched }) => (
-					<Form>
-						<AutoSubmitToken />
-						<atoms.Typography type='h4'>
-							Idea Info
-							<hr noshade="false" style={{margin: "1rem auto 2rem"}}/>
-						</atoms.Typography>
+  const compareValuesToInitial = (values) => {
```
Comment (chungthuang) [inline apps/ideaspace/src/components/common/IdeaForm/IdeaForm.js:28]: Can you give an example of what are we comparing?

File: apps/ideaspace/src/components/common/IdeaForm/IdeaForm.js [hunk 4]
```diff
@@ -7,227 +7,236 @@ import SubmissionButton from './SubmissionButton';
...
+                    />
+                    <atoms.Radio
+                      onChange={(e, label) =>
+                        setFieldValue('involveLevel', label)
+                      }
+                      label='I want to "own" this idea only after it is approved as a project'
+                    />
+                    <atoms.Radio
+                      onChange={(e, label) =>
+                        setFieldValue('involveLevel', label)
```
Comment (chungthuang) [inline apps/ideaspace/src/components/common/IdeaForm/IdeaForm.js:183]: Can we set field to "highly", so we don't need this conversion https://github.com/dev-launchers/dev-launchers-platform/pull/1944/files#r1744700300?

File: packages/UI/src/components/atoms/Radio/StyledRadio.ts (hunk 1 above)
Comment (chadcrotchett) [reply packages/UI/src/components/atoms/Radio/StyledRadio.ts:25]: hard-coded pixel width was causing the radio 'circles' to be 'ovals' that were stretched out

File: apps/ideaspace/src/components/modules/SubmissionForm/SubmissionForm.js (hunk 2 above)
Comment (dbradham) [reply apps/ideaspace/src/components/modules/SubmissionForm/SubmissionForm.js:35]: @chungthuang  We can, but seems out of scope for this frontend change. We would still need to map the values to an enum.

Do you have a solution in mind that avoids making a comparison like this to get a value to save to the database?

Comment (dbradham) [reply apps/ideaspace/src/components/modules/SubmissionForm/SubmissionForm.js]: @chungthuang just saw your other comment and updated this

File: apps/ideaspace/src/components/common/IdeaForm/IdeaForm.js (hunk 4 above)
Comment (dbradham) [reply apps/ideaspace/src/components/common/IdeaForm/IdeaForm.js]: @chungthuang yes thanks for the recommendation here, this is complete.

File: apps/ideaspace/src/components/common/IdeaForm/IdeaForm.js (hunk 3 above)
Comment (dbradham) [reply apps/ideaspace/src/components/common/IdeaForm/IdeaForm.js:28]: @chungthuang this is to check if any form values have been updated.

File: apps/ideaspace/src/components/modules/WorkshoppingPage/CommentsForm/SingleComment.js [hunk 5]
```diff
@@ -139,12 +139,12 @@ function SingleCommentComponent(props) {
             </div>
           </SingleCommentContent>
           <SingleCommentButtons style={{ display: 'flex' }}>
-            <LikeButton
+            {/* <LikeButton
```
Comment (chungthuang) [inline apps/ideaspace/src/components/modules/WorkshoppingPage/CommentsForm/SingleComment.js:142]: Why are we commenting this out?

**Review by chungthuang — APPROVED** [review]

**Pull request merged**